sources/index.spec.js

```diff
@@ -184,8 +180,8 @@ describe('chai-style', () => {
     })
   })
 
-  describe('Different order of values', () => {
-    it('test box-shadow', () => {
+  describe('Differents order of pixels values', () => {
+    it('box-shadow test', () => {
```

Comment:
how in line 25, the css applied is "0 0 10px red", in this spec, maybe be better write something very different in order, like "#f00 0 0 10px" , to be explicit, when anyone read this spec (legibility)